README.md

```diff
@@ -15,45 +17,89 @@ The code consists of a patch series morphed and rebased onto every new Chrome re
 
 [Browser → Naïve (client)] ⟶ Censor ⟶ [Frontend → Naïve (server)] ⟶ Internet
 
-NaïveProxy uses Chrome's network stack. The traffic behavior intercepted by the censor is identical to regular HTTP/2 traffic between Chrome and standard Frontend (e.g. Caddy, HAProxy).
+NaïveProxy uses Chrome's network stack. Thus, the traffic behavior intercepted by the censor is identical to regular HTTP/2 traffic between Chrome and standard Frontend (e.g., Caddy, HAProxy).
```

Comment:
Comma after e.g. is optional https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e.